Framework/DataHandling/src/LoadNexusLogs.cpp

```diff
@@ -654,5 +656,33 @@ LoadNexusLogs::createTimeSeries(::NeXus::File &file,
   }
 }
 
+/**
+  * Checks whether the specified character is invalid or a control
+  * character. If it is invalid (i.e. negative) or a control character
+  * the method returns true. If it is valid and not a control character
+  * it returns false. Additionally if the character is invalid is
+  * logs a warning with the property name so users are aware.
+  *
+  * @param c :: Character to check
+  * @param propName :: The name of the property currently being checked for
+  *logging
+  * @param log :: Reference to logger to print out to
+  * @return :: True if control character OR invalid. Else False
+  */
+bool LoadNexusLogs::isControlValue(const char &c, const std::string &propName,
+                                   Kernel::Logger &log) {
+  // Have to check it falls within range accepted by c style check
+  if (c <= -1) {
```

Comment:
So I double checked this as I was unsure what the compiler would do. According to the C++ standard this is up to the compiler:
http://en.cppreference.com/w/cpp/language/implicit_conversion#Numeric_promotions
So the compiler could compare `char` types or `int` types (using promotion).

Checking the compiler output confirms my suspicions - both the compiler chosen type and static_cast types are treated identically because of this:
https://godbolt.org/g/7lFs4p

For this reason I think a static_cast wouldn't be appropriate here as it implies we can check for a range larger than char (which is untrue) and it may force the compiler to use `int` types when a `char` type is appropriate